src/Dialect/Krnl/KrnlOps.cpp

```diff
@@ -812,6 +814,50 @@ MutableOperandRange KrnlSpecializedKernel::getLoopRefs() {
   return getLoopsMutable();
 }
 
+ArrayRef<char> getRawData(KrnlGlobalOp &op) {
```

Comment:
Again, we have some duplicated code here. To avoid that, this function can be a function in `ConstantOpInterface` and we can name it `getValueBuffer()`, and `getBuffer()` can be renamed to `getOutputBuffer()`.